wrapanapi/systems/rhevm.py

```diff
@@ -71,6 +71,10 @@ def _get_network(self, network_name):
     def get_nics(self):
         return self.api.nics_service().list()
 
+    def get_vnic_profiles(self):
+        """ Get vnic_profiles of the VM/template """
+        return [nic.vnic_profile for nic in self.get_nics()]
```

Comment:
I'd turn it into property - vnic_profiles

Reply:
I thought the same thing @izapolsk, but `get_nics` above establishes a pattern, I think we should be consistent.  Consistent either way though - either they should all be properties, or `get_<thing>` methods.

Reply:
I'd better use first but you're right code has to be consistent